README.md

```diff
@@ -1,3 +1,14 @@
+Some assumptions/decisions made:
+    1. For modularizing the AccountTrigger code, the assignment isn't to extend the TriggerHandler, but to simply pull most of the logic out of the AccountTrigger.
+    2. Since that leaves the AccountTrigger simply calling methods in the Helper, I'm combining the 3 identical If Statements into 1, reducing code and operation.  Not sure if that's good practice or not, just making a call on this.
+    3. For the trigger: Append Stage changes in Opportunity Description, which was originally After Update, this makes no sense.  I moved that to before update, which is a more proper position for this kind of update (unless I'm not understanding some requirement).
+    4. In Part 3, I have used AnotherOpportunityTrigger.trigger as the sole operating trigger for the org.  I've made this move in stages, which can be traced in my (likely too many) commits.  In presenting final products, I've adjusted all commenting and removed previous code entirely from this file rather than merely comment that out.
+    5. For the two before delete pieces of logic.  To combine these, I chose to simply keep the broader logic, which preventing deleting closed cases, as the validation to prevent certain Closed Won cases from being deleted was already accomplished with the first trigger.  
+        Was unsure about this, as best practice might be to code the other piece in as well in case the first validation was changed in the future????
```

Comment:
You could maybe document the dual requested features. Probably also work with stakeholders to determine the desired behavior. Usually these sorts of requests came in at different times and/or were written by different people.